src/app/backend/replicasetlist.go

```diff
@@ -56,47 +58,132 @@ type ReplicaSet struct {
 	CreationTime unversioned.Time `json:"creationTime"`
 
 	// Internal endpoints of all Kubernetes services have the same label selector as this Replica Set.
+	// Endpoint is DNS name merged with ports.
 	InternalEndpoints []string `json:"internalEndpoints"`
 
 	// External endpoints of all Kubernetes services have the same label selector as this Replica Set.
+	// Endpoint is external IP address name merged with ports.
 	ExternalEndpoints []string `json:"externalEndpoints"`
 }
 
 // Returns a list of all Replica Sets in the cluster.
 func GetReplicaSetList(client *client.Client) (*ReplicaSetList, error) {
-	list, err := client.ReplicationControllers(api.NamespaceAll).
+	replicaSets, err := client.ReplicationControllers(api.NamespaceAll).
 		List(labels.Everything(), fields.Everything())
 
 	if err != nil {
 		return nil, err
 	}
 
+	services, err := client.Services(api.NamespaceAll).
+		List(labels.Everything(), fields.Everything())
+
+	if err != nil {
+		return nil, err
+	}
+
+	return getReplicaSetList(replicaSets.Items, services.Items), nil
+}
+
+// Returns a list of all Replica Sets in the cluster, based on all Kubernetes Replica Set and
```

Comment:
Could you explain in the comment a bit more what this method does? What are the arguments for?